packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1355,15 +1355,14 @@ model = SwarmFormerModel.from_pretrained("${model.id}")
 
 const mlx_unknown = (model: ModelData): string[] => [
 	`# Download the model from the Hub
-pip install huggingface_hub hf_transfer
+pip install huggingface_hub[hf_xet]
```

Comment:
cc: @jsulz - should we put any other env variables here? IIRC no, but wondering if you'd recommend any!

Reply:
put this here: https://github.com/huggingface/huggingface.js/pull/1463/commits/3c5ab825a4cef59d89c9b75e03b5c21387a71704

Reply:
Updated here: https://github.com/huggingface/huggingface.js/pull/1463/commits/3c5ab825a4cef59d89c9b75e03b5c21387a71704